tools/system_libs.py

```diff
@@ -41,7 +41,7 @@ def read_symbols(path, exclude=None):
       symbols = filter(lambda symbol: symbol not in exclude, symbols)
     return set(symbols)
 
-  default_opts = []
+  default_opts = ['-Werror']
```

Comment:
I believe @juj added it after cleaning up all known warnings in system libs, to avoid new warnings appearing in the future (the warnings make the first-run experience quite bad; and clang updates often add new warnings).

I guess you're seeing a warning in a wasm system lib, which @juj didn't test on? We should investigate fixing the warning in the source code if possible, or if somehow not, then we could consider disabling `-Werror` on certain files perhaps?